src/OrbitGl/App.cpp

```diff
@@ -863,24 +864,28 @@ bool OrbitApp::StartCapture() {
   selected_functions.insert(selected_functions.end(), orbit_functions.begin(),
                             orbit_functions.end());
 
+  UserDefinedCaptureData user_defined_capture_data = data_manager_->user_defined_capture_data();
+
   absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions_map;
+  absl::flat_hash_set<uint64_t> frame_track_function_ids;
+  uint64_t function_id = 1;
```

Comment:
Maybe add a comment, that you are starting to compute the function_id's here. Also that 0 is a special case.

Reply:
Done. and I it is now starting with 7 - so that it is not confused with an index :)